9.2/run-postgresql.sh

```diff
@@ -90,12 +107,16 @@ function initialize_database() {
 
 		# Allow connections from all hosts.
 		host all all all md5
+
+		# Allow replication connections from all hosts.
+		host replication all all md5
 	EOF
 
 	pg_ctl -w start
 	createuser "$POSTGRESQL_USER"
 	createdb --owner="$POSTGRESQL_USER" "$POSTGRESQL_DATABASE"
 	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_PASSWORD}';"
+	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH REPLICATION ;"
```

Comment:
OK, that sounds like an ideal solution (hope it won't take too long to implement). I'll open an issue for that after this PR is merged.